XamarinCommunityToolkit.UnitTests/Xamarin.CommunityToolkit.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
@VladislavAntonyuk is there any reason you added this as part of your WPF and GTK PR?

Reply:
Snackbar unit tests are failing with netcoreapp3.1 because it tries to run Snackbar for WPF